telegram/_chatadministratorrights.py

```diff
@@ -32,6 +32,8 @@ class ChatAdministratorRights(TelegramObject):
     :attr:`can_post_messages`, :attr:`can_edit_messages`, :attr:`can_pin_messages`,
     :attr:`can_manage_topics` are equal.
 
+    .. versionchanged:: 20.0
```

Comment:
you'll also have to explain *what* changed ;) see https://github.com/python-telegram-bot/python-telegram-bot/blob/8dead11a25f4b6888bad4fee356b45a91b1f4565/telegram/_webhookinfo.py#L42-L44